proposal/benchen216/README.rst

```diff
@@ -0,0 +1,64 @@
+==========================================
+Proposal for logic gate circuit simulation
+==========================================
+
+Basic information
+=================
+In recent days, FPGA SoC and some kinds of SoC can be use as AI or deep learning accessor,
+also 5g will use FPGA as RFSoc to extend the radio transmission.
+
+project URL:
+
+https://github.com/benchen216/logic-gate-sim
+
+
+Perspective users
+=================
+
+1. The student that feel verilog is so difficult.
+2. The people want to run simple logic circuit.
+
+System architecture
+===================
+
+Data Flow
+
+1. read in logic gate circuit syntax
+2. construct the circuit and fill to data type
+3. read in the input to run the circuit
+4. output the result
+
+API description
+===============
+
+1. logic gate circuit
```

Comment:
I think the list here is what the API is about, rather than the API itself.  Could you add the envisioned API or the look-and-feel?  Even if it's just a guess, it will provide you an idea how it could work.